ui/src/timeMachine/components/Queries.tsx

```diff
@@ -134,11 +147,13 @@ const mstp = (state: AppState) => {
     activeQuery,
     autoRefresh,
     isInCheckOverlay: getIsInCheckOverlay(state),
+    inVEOMode: isInVEOMode(state),
```

Comment:
So... I think this state is the same as checking with the router to see if you are in the edit cell path. before querying you could check if you are in VEO path, and not update if so.

Reply:
That's a really good point that I hadn't considered